x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_trend_graph.tsx

```diff
@@ -72,7 +72,7 @@ const AccountDropDown = ({
   options: Array<{ value: string; label: string }>;
 }) => (
   <EuiComboBox
-    style={{ width: 320 }}
+    css={{ width: 320 }}
```

Comment:
Is this responsive?

If it's not we should raise this in a follow-up issue to use breakpoints if we can.